include/git2/common.h

```diff
@@ -305,6 +306,9 @@ typedef enum {
  *		>
  *		> - `ciphers` is the list of ciphers that are eanbled.
  *
+ *	* opts(GIT_OPT_ENABLE_OFS_DELTA, int enabled)
+ *
+ *		> Enable or disable ofs_delta capability.
```

Comment:
I think that this name is scoped nicely to (1), _never create packfiles with offset deltas_.  I also think that this is the sensible scoping for this feature.  As it happens, we will never write packfiles with offset deltas (this is still surprising to me!) so that's an easy win for that part of this support.  😀